src/SpeciesElements.f90

```diff
@@ -0,0 +1,167 @@
+
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief Utility module for coverting different forms of elements and isotopes
+!>
+!> This package provides an interface to convert a chemical formula into an 
+!> input array for coupling with Thermochimica
+!>
+!> @par Module Dependencies
+!>  - @ref IntrType "IntrType": @copybrief IntrType
+!>  - @ref Strings "Strings": @copybrief Strings
+!>  - @ref ElementsIsotopes "ElementsIsotopes": @copybrief ElementsIsotopes
+!>
+!> @author Zack Taylor
+!>   @date 01/24/19
+!>
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+MODULE SpeciesElements
+  USE IntrType
+  USE Strings
+  USE ElementsIsotopes
+  IMPLICIT NONE
+  PRIVATE
+
+  ! Member
+  PUBLIC :: SpeciesElementsType
+
+  !> Type that converts a given species chemical formula and molar amount
+  !! to an array of molar amounts ordered by atomic number 
+  TYPE :: SpeciesElementsType 
+
+  !List of type bound procedures
+  CONTAINS
+    PROCEDURE,PASS :: getElementArray
+    PROCEDURE,PASS,PRIVATE :: isChar
+    PROCEDURE,PASS,PRIVATE :: isCharCap
+    PROCEDURE,PASS,PRIVATE :: isCharLow
+  ENDTYPE SpeciesElementsType
```

Comment:
A couple high level comments on this:
1.  The `TYPE` isn't really doing anything at the moment.  I'd rather just have `getElementArray` be a standalone subroutine that can be called.  However, I know you were just doing the same thing as in `ElementsIsotopes`... @stimpsonsg what are your thoughts?  I'd rather not have to instantiate an object just to call a procedure, personally.
2.  The `isChar*` subroutines really should probably live as separate routines in `IO_Strings.f90`.  In fact, double check that module to see if there are functions there that could accomplish this purpose.